packages/lib/src/components/buttons/InfoButtonComponent.wc.svelte

```diff
@@ -33,28 +41,45 @@
         tooltipOpen = false;
     };
 
-    const displayQueryInfo = (): void => {
+    const displayQueryInfo = (queryItem?: AstBottomLayerValue): void => {
         if (showQuery) {
-            message =
-                getHumanReadableQuery().length > 0
-                    ? [getHumanReadableQuery()]
-                    : [noQueryMessage];
+             if (onlyChildInfo) {
+                let childMessage =
+                    buildHumanReadableRecursively(
+                    returnNestedValues(queryItem), "");
+                message = childMessage.length > 0
+                    ? childMessage : noQueryMessage;
```

Comment:
`childMessage` and `noQueryMessage` should be in an array. That way don't need the conditional in line 76.